frontend/src/CalendarCell/CalendarCell.tsx

```diff
@@ -0,0 +1,20 @@
+import classes from './CalendarCell.module.css';
+
+type CalendarCellProps = {
+  date?: string
+};
+
+function CalendarCell(props: CalendarCellProps) {
+  return (
+    <div className={classes.calendarCellWrapper}>
+        <div className={classes.calendarCellHeader}>
+            <h1 className={classes.dateElement}>{props.date ? props.date : ''}</h1>
```

Comment:
there's a lot of `<h1>`s going on on this page, which looks fine style-wise but might be confusing to a screen reader. i recommend keeping the topmost and most important `<h1>` and then making the others `<p>` tags that are big and bold using css.